config/crd/bases/infrastructure.cluster.x-k8s.io_ibmpowervsclusters.yaml

```diff
@@ -18,7 +18,7 @@ spec:
   - name: v1alpha4
     schema:
       openAPIV3Schema:
-        description: IBMPowerVSCluster is the Schema for the ibmpowervsclusters API
+        description: IBMPowerVSCluster is the Schema for the ibmpowervsclusters API.
```

Comment:
wondering how it listing didn't fail before?!

Reply:
The changes were added as part of `make manifests` target, so I am assuming we need to add missing `verify-manifests` target to Makefile.

Reply:
lets create a task for the same.